### PR TITLE
#140 shopping cart

### PR DESCRIPTION
**Pull request by J0SAL**
Fixes for #140

### Changes
1. Background color of the shopping page changed to `grey`.
2. Updated text heading from "`Hello, Your shopping basket"` to `"Shopping Cart"`.
3. Added a `Deselect all items' option` which removes all the selected items.
4. Adjusted the `style and width` of item headings.
5. Added Rupee(`₹`) symbol in prices.

### Updated UI
image

### CODE REVIEW

**Comment by Kanika637** [comment]
Hey @J0SAL, You should first make the issue assigned to yourself and then make the PR. Kindly don't repeat this activity next time.

File: src/Checkout.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .checkout {
   display: flex;
   padding: 20px;
-  background-color: white;
+  background-color: grey;
```
Comment (Kanika637) [inline src/Checkout.css:4]:
```suggestion
  background-color: #f0ecec;
```

**Review by Kanika637 — CHANGES_REQUESTED** [review]
Bold the heading of "Subtotal" like this 
image

**Comment by J0SAL** [comment]
> Hey @J0SAL, You should first make the issue assigned to yourself and then make the PR. Kindly don't repeat this activity next time.

Thanks for pointing out my mistake. I'll remember to follow the guidelines henceforth.

**Comment by J0SAL** [comment]
I have updated the required changes. @Kanika637 Could you please review the same.

**Comment by Kanika637** [comment]
Can you now provide the final screenshots @J0SAL?

**Comment by J0SAL** [comment]
Here's the screenshot from the latest changes. @Kanika637 
image

**Review by Kanika637 — APPROVED** [review]
LGTM🚀
Great Work
Don't forget to give a ⭐ to the repo, if you learned something

**Pull request merged**